### ISSUE TITLE
Avoid default exports for locale files

### ISSUE THREAD

**Issue by krechtern** [t1m0n/air-datepicker#586]
I'm currently trying to use this library inside an angular library. When building the library the locale import will interpreted in a way that I receive `{default: {…}, __esModule: true}` instead of the locale object. This seems to be a common problem as described here: https://esbuild.github.io/content-types/#default-interop

Maybe it would be a good approach to avoid the default exports for the locale files, so this problem does not occur for other users too.

**Comment by ConnectGrid** [comment]
Agreed. The way that the locale modules are written is really complicating the usage of this library. If they exported something like "localeEn" by default, then it may be easier to use in a dynamic situation?
